paws_collector.js

```diff
@@ -14,13 +14,64 @@ const async = require('async');
 const debug = require('debug')('index');
 const AWS = require('aws-sdk');
 const fs = require('fs');
+const moment = require('moment');
 
 const AlAwsCollector = require('@alertlogic/al-aws-collector-js').AlAwsCollector;
 const m_packageJson = require('./package.json');
 
 const CREDS_FILE_PATH = '/tmp/paws_creds.json';
 var PAWS_DECRYPTED_CREDS = null;
 
+function calcNextCollectionInterval(strategy, curUntil, pollInterval) {
```

Comment:
I'd be nice to add TS format option to indicate which Ts is used as input/output.
It'd be enough to support ISO8601 and Unix formats for now.

Reply:
addressed